logsearch-platform-jobs.yml

```
@@ -82,15 +82,27 @@ instance_groups:
     properties:
       elasticsearch_config:
         app_index_prefix: logs-platform
-        templates:
+        component_templates:
         - shards-and-replicas: /var/vcap/jobs/elasticsearch_config/index-templates/shards-and-replicas.json
         - index-settings: /var/vcap/jobs/elasticsearch_config/index-templates/index-settings.json
         - index-mappings: /var/vcap/jobs/elasticsearch_config/index-templates/index-mappings.json
+        - c-index-mappings-lfc: /var/vcap/jobs/elasticsearch-config-lfc/index-mappings.json
+        - c-index-mappings-app-lfc: /var/vcap/jobs/elasticsearch-config-lfc/index-mappings-app.json
+        - c-index-mappings-platform-lfc: /var/vcap/jobs/elasticsearch-config-lfc/index-mappings-platform.json
+        index_templates:
```

Matching changes to customer-facing logs:

```suggestion
        - index-mappings-lfc: /var/vcap/jobs/elasticsearch-config-lfc/component-index-mappings.json
        - index-mappings-app-lfc: /var/vcap/jobs/elasticsearch-config-lfc/component-index-mappings-app.json
        - index-mappings-platform-lfc: /var/vcap/jobs/elasticsearch-config-lfc/component-index-mappings-platform.json
        index_templates:
```